pkg/monitoring/metrics/template-validator/metrics.go

```diff
@@ -9,8 +9,3 @@ func SetupMetrics() error {
 		templateMetrics,
 	)
 }
-
-// ListMetrics registered prometheus metrics
-func ListMetrics() []operatormetrics.Metric {
-	return operatormetrics.ListMetrics()
-}
```

Comment:
@machadovilaca , I've found this unused function and removed it. Is this a bug, that it is not used anywhere?

Reply:
In usually call these functions the tooling to generate the docs, but when setting up multiple metrics, in the registry we only need to list from one. That's why the ListMetrics from metrics/ssp-operator is used but not this one

Reply:
So, this was not a bug, that the function was not called?

Reply:
No, we can remove it. On a future release of the operator-observability library, it will support different registries, and then we will improve the docs to setup and list the metrics separately.